docs/guides/interfaces.rst

```diff
@@ -38,28 +34,32 @@ However, using abstract classes requires using inheritance, which sometimes is u
+
+.. list-table::
+    :align: left
+
+    * - :class:`CircuitRunner <orquestra.quantum.api.circuit_runner.CircuitRunner>` (:ref:`guide <backends_guide>`)
+    * - :class:`WavefunctionSimulator <orquestra.quantum.api.wavefunction_simulator.WavefunctionSimulator>` (:ref:`guide <backends_guide>`)
+    * - :class:`Optimizer <orquestra.opt.api.Optimizer>` (:ref:`guide <optimizers_guide>`)
+    * - :class:`CostFunction <orquestra.opt.api.CostFunction>`
+    * - :class:`Ansatz <orquestra.vqa.api.ansatz.Ansatz>` (:ref:`guide <ansatzes_guide>`)
+    * - :class:`Estimators <orquestra.quantum.api.estimation.EstimateExpectationValues>` (:ref:`guide <estimators_guide>`)
 
 
 Passing parameters
 ------------------
 
-You might wonder - what if I have two optimizers which take different sets of parameters? For example, you might want to use `L-BFGS-B <https://docs.scipy.org/doc/scipy/reference/optimize.minimize-lbfgsb.html#optimize-minimize-lbfgsb>`_ and `COBYLA <https://docs.scipy.org/doc/scipy/reference/optimize.minimize-cobyla.html#optimize-minimize-cobyla>`_ from `scipy`. This is how you would use them in scipy: 
+You might wonder - what if I have two optimizers which take different sets of parameters? For example, you might want to use `L-BFGS-B <https://docs.scipy.org/doc/scipy/reference/optimize.minimize-lbfgsb.html#optimize-minimize-lbfgsb>`_ and `COBYLA <https://docs.scipy.org/doc/scipy/reference/optimize.minimize-cobyla.html#optimize-minimize-cobyla>`_ from ``scipy``. This is how you would use them in scipy: 
 
 .. literalinclude:: ../examples/guides/interfaces_guide.py
     :start-after: >> Guide code snippet: script showing scipy optimizers
     :lines: 2-3
     :language: python
 
 
-`L-BFGS-B` doesn't support argument called `constraints` and `COBYLA` doesn't support `bounds`. So how can you make them fit the same interface, if they have different inputs?
+``L-BFGS-B``` doesn't support argument called ``constraints`` and ``COBYLA`` doesn't support ``bounds``. So how can you make them fit the same interface, if they have different inputs?
 
-In the case of optimizers, Orquestra solves this problem by moving differing parameters into object initialization. Constructed optimizers provide :meth:`minimize <orquestra.opt.api.Optimizer.minimize>` method that always accepts only :func:`create_cost_function <orquestra.vqa.cost_function.cost_function.create_cost_function>` and `initial_params` arguments. Once constructed, optimizers are fully interchangeable.
+In the case of optimizers, Orquestra solves this problem by moving differing parameters into object initialization. Constructed optimizers provide :meth:`minimize <orquestra.opt.api.Optimizer.minimize>` method that always accepts only :func:`create_cost_function <orquestra.vqa.cost_function.cost_function.create_cost_function>` and ``initial_param`s`` arguments. Once constructed, optimizers are fully interchangeable.
```

Comment:
I think something is wrong here – the name of the arg is `initial_params`, not `initial_param`?

Reply:
An extra backtick snuck in here, now fixed in faec13157e2adf175e2306705f526a4ec977cca2